cli/cli/Services/BeamoLocalSystem_Docker.cs

```diff
@@ -320,39 +320,79 @@ public Task<string> PullAndCreateImage(string publicImageName, Action<float> pro
 		return _client.PullAndCreateImage(publicImageName, progressUpdateHandler);
 	}
 
+	private static List<string> ParseDockerfile(string dockerFilePath)
+	{
+		var paths = new List<string>();
+
+		var fileContents = File.ReadAllLines(dockerFilePath);
+
+		foreach (string line in fileContents)
+		{
+			if (line.StartsWith("COPY"))
+			{
+				var parts = line.Split(" ");

```

Comment:
use remove empty entries here, and check if this array has COPY SOURCE DESTINATION, 3 elements, then it's correct. And throw specific exceptions that we care about in the GetAttributes call